security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-release-notes"]
+= {cert-manager-operator} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-release-notes
+
+toc::[]
+
+The {cert-manager-operator} is a cluster-wide service that provides application certificate lifecycle management.
+
+These release notes track the development of {cert-manager-operator}.
+
+:FeatureName: The {cert-manager-operator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
+
+[id="cert-manager-operator-release-notes-1.7.1"]
+== Release notes for {cert-manager-operator} 1.7.1 (Technology Preview)
+
+// TODO: Update if needed
+Issued: 2022-03-29
+
+The following advisory is available for the {cert-manager-operator} 1.7.1:
+
+// TODO: Replace link and name
+* link:https://access.redhat.com/errata/RHEA-TODO[RHEA-TODO - TODO]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.7/#v1-7-1[upstream cert-manager v1.7.1 release notes].
+
+[id="cert-manager-operator-1.7.1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* This is the initial, Technology Preview release of the {cert-manager-operator}.
+
+////
+// No bug fixes in the initial release
+[id="cert-manager-operator-1.7.1-bug-fixes"]
+=== Bug fixes
+
+* TODO
+////
+
+[id="cert-manager-operator-1.7.1-known-issues"]
+=== Known issues
+
+* Using `Route` objects is not fully supported. Currently, {cert-manager-operator} integrates with `Route` objects by creating `Ingress` objects through the Ingress Controller. (link:https://issues.redhat.com/projects/CM/issues/CM-16[*CM-16*])
+// TODO: Make sure that CM is a public project
```

Comment:
@slaskawi Let me know once this is public - want to make sure it is before I merged these docs.